keyboards/maxr1998/pulse4k/readme.md

```diff
@@ -1,6 +1,8 @@
 # Pulse 4k
 A four-key macropad with two rotary encoders, developed by Maxr1998, [fully open-source](https://github.com/Maxr1998/Pulse_4k).
 
+<p align="center"><img width="60%" src="https://i.imgur.com/g1QyqxS.jpg"></p>
```

Comment:
`![image](https://i.imgur.com/g1QyqxS.jpg)` would work too.

![image](https://i.imgur.com/g1QyqxS.jpg)


```suggestion
![image](https://i.imgur.com/g1QyqxS.jpg)
```

Reply:
Yes, but then it'd get rather big. The HTML code fixes that (note the width attribute).